src/components/ErrorBoundary/index.tsx

```diff
@@ -1,63 +0,0 @@
-import React, { FC } from 'react';
```

Comment:
Why do we not need the ErrorBoundary component anymore?

Reply:
We removed it because you can handle it in the Router, but that a good question, maybe we should keep this component even if not used 🤔